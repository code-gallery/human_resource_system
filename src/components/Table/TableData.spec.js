import React from 'react'
import renderer from 'react-test-renderer'
import { TableData } from './index'

describe('<TableData />', () => {
  it('renders correctly', () => {
    const tree = renderer.create(
      <TableData>React</TableData>
    ).toJSON()

    expect(tree).toMatchSnapshot()
  })
})
