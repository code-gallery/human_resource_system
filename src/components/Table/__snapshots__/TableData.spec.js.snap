// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TableData /> renders correctly 1`] = `
<td
  className="TableData"
>
  React
</td>
`;
